README.md

```diff
@@ -14,45 +14,44 @@ Also the RNE team runs and maintains this project, so if you need any help devel
 
 ### Run it locally
 
-1. Clone the project
+1. Install [Expo CLI](https://docs.expo.io/versions/latest/workflow/expo-cli/)
 
 ```
-git clone https://github.com/react-native-training/react-native-elements-app.git
+[sudo] npm install -g expo-cli
 ```
 
-2. Install dependencies
+*If permissions errors then please use `--unsafe-perm=true` flag too [npm/npm#16766](https://github.com/npm/npm/issues/16766)*
 
-```
-cd react-native-elements-app
+2. Clone the project
 
-# Using npm or yarn
-yarn || npm install
+```
+git clone https://github.com/react-native-training/react-native-elements-app.git
 ```
 
-3. Run the mobile app (uses [Expo](https://expo.io/learn))
+3. Install dependencies
 
 ```
-# Install the expo-cli
-npm install -g expo-cli
+cd react-native-elements-app
 
-yarn native || npm run native
+# Using npm or yarn
+yarn || npm install
 ```
 
-4. Run the web app (uses [CRA](https://facebook.github.io/create-react-app/docs/getting-started))
+4. Run the cross-platform app (uses [Expo](https://expo.io/learn))
 
 ```
 yarn web || npm run web
```

Comment:
Think this should be

```suggestion
yarn start || npm start
```

Reply:
@iRoachie  `yarn web` starts up Metro bundler for native platform and webpack for web but `yarn start` - just Metro, without webpack.

Reply:
We can tuning package.json's scripts and README inside `update` branch I think...

Reply:
Yea but `yarn web` also opens the app in a browser. I think we should leave it as `yarn start`, the underlying `expo start` command shows you steps for running on web/ios/android.

yarn start
- press `i` (opens ios)
- press `a` (opens android)
- press `w` (opens web)